prime-router/src/test/resources/fhir_sender_transforms/test_extension_schema.yml

```
@@ -0,0 +1,14 @@
+#extends: test_base_schema
```

I think we need to uncomment this for the test to do what you meant for it to do.
```suggestion
extends: test_base_schema
```